mkdud

```diff
@@ -513,6 +515,31 @@ Adding RPMs notes:
     you must configure the build service access via .oscrc in your HOME directory for this
     to work.
 
+Distribution (product) name notes:
```

Comment:
Maybe could be interesting also to add this documentation to the README.md document